PeakLocalisations/Localisation/Blocks/Trends.xcstrings

```diff
@@ -15,6 +15,12 @@
             "state" : "translated",
             "value" : "Stay on track and build on your past performance."
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Mira las tendencias en su progreso."
```

Comment:
A subtle thing, this is mixing the informal and formal 'you' (tú/usted). It should either be "Mira... en tu..." or "Mire... en su...".
Also it would be best to maintain the same form throughout the app but it's not a huge deal.